### PR TITLE
zoekt: init at unstable-2021-03-17

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/zoekt/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+buildGoModule {
```
Comment (SuperSandro2000) [inline pkgs/tools/text/zoekt/default.nix:6]:
```suggestion
}:

buildGoModule {
```

File: pkgs/tools/text/zoekt/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    repo = "zoekt";
+    rev = "d92b3b80e582e735b2459413ee7d9dbbf294d629";
+    sha256 = "JdORh6bRdHsAYwsmdKY0OUavXfu3HsPQFkQjRBkcMBo=";
+  };
+
+  vendorSha256 = "d+Xvl6fleMO0frP9qr5tZgkzsnH5lPELwmEQEspD22M=";
+
+  checkInputs = [
+    git
+  ];
```
Comment (SuperSandro2000) [inline pkgs/tools/text/zoekt/default.nix:21]: That should be part of the buildGoModule env.

Comment (cole-h) [reply pkgs/tools/text/zoekt/default.nix:21]: Unfortunately, it's not:

```
=== RUN   TestFullAndShortRefNames
    tree_test.go:434: createMultibranchRepo: execution error: exit status 127, output + mkdir repo
        + cd repo
        + git init
        /bin/sh: git: not found
--- FAIL: TestFullAndShortRefNames (0.00s)
```

Comment (SuperSandro2000) [reply pkgs/tools/text/zoekt/default.nix:21]: Maybe its only for the buildPhase,

**Pull request merged**